frontend/src/app/user-interface/burger-menu/burger-menu.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { MatMenuModule, MatButtonModule, MatIconModule } from '@angular/material';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { MenuItem } from './MenuItem';
+
+@Component({
+  selector: 'app-burger-menu',
+  templateUrl: './burger-menu.component.html',
+  styleUrls: ['./burger-menu.component.scss']
+})
+export class BurgerMenuComponent implements OnInit {
+  lastSelected:string = "";
+  menuOptions:Array<MenuItem> = [
+        {elementSelector: 'wc', text: 'WC', icon: 'wc'},
+        {elementSelector: 'office', text: 'BIURA', icon: 'business_center'},
+        {elementSelector: 'room', text: 'SALE', icon: 'meeting_room'},
+        {elementSelector: 'printer', text: 'DRUKARKI', icon: 'print'},
+        {elementSelector: 'elevator', text: 'WINDY', icon: 'arrow_upward'},
+        {elementSelector: 'kitchen', text: 'KUCHNIE', icon: 'restaurant'},
+        {elementSelector: 'reception', text: 'RECEPCJA', icon: 'room'},
+  ];
+  constructor() { }
+
+  ngOnInit() {
+  }
+    onClick(elementSelector:string){
+        if(this.lastSelected)
+        {
+            var prevElementList = document.querySelectorAll(`[id^=${CSS.escape(this.lastSelected)}]`);
```

Comment:
funkcja CSS.escape, zastanawiam się czy na pewno jest potrzebna teraz. A jeśli jest to ryzyko na IE się pojawia, bo nie jest wspierane